wezterm-gui/src/overlay/selector.rs

```diff
@@ -178,25 +189,46 @@ impl SelectorState {
     }
 
     fn run_loop(&mut self, term: &mut TermWizTerminal) -> anyhow::Result<()> {
+        let max_items = self.max_items;
+        let alphabet = self.args.alphabet.to_lowercase();
+        let alphabet_has_j = alphabet.contains("j");
+        let alphabet_has_k = alphabet.contains("k");
+        let labels = compute_labels_for_alphabet(&alphabet, max_items + 1);
+
         while let Ok(Some(event)) = term.poll_input(None) {
             match event {
                 InputEvent::Key(KeyEvent {
                     key: KeyCode::Char(c),
-                    ..
-                }) if !self.filtering && c >= '1' && c <= '9' => {
-                    if self.launch(self.top_row + (c as u32 - '1' as u32) as usize) {
-                        break;
+                    modifiers: Modifiers::NONE,
+                }) if !self.filtering && alphabet.contains(c) => {
+                    self.selection.push(c);
+                    if let Some(pos) = labels.iter().position(|x| *x == self.selection) {
+                        if pos as usize <= max_items && self.launch(self.top_row + pos as usize) {
+                            break;
+                        }
                     }
                 }
                 InputEvent::Key(KeyEvent {
                     key: KeyCode::Char('j'),
                     ..
-                }) if !self.filtering => {
+                }) if !self.filtering && !alphabet_has_j => {
                     self.move_down();
                 }
                 InputEvent::Key(KeyEvent {
                     key: KeyCode::Char('k'),
                     ..
+                }) if !self.filtering && !alphabet_has_k => {
+                    self.move_up();
+                }
+                InputEvent::Key(KeyEvent {
+                    key: KeyCode::Char('J'),
+                    ..
```

Comment:
I'm not sure about this uppercase version of the movement keys.
In the launcher overlay we have CTRL-J/K as alternatives to CTRL-N/P; I'd be cool with having those here instead of this.

https://github.com/wez/wezterm/blob/99c9613966a73d5388344fd0df68d5977b493c7c/wezterm-gui/src/overlay/launcher.rs#L482-L493

Reply:
I have done it like this now.